lib_runtime/mirage_pp.ml

```diff
@@ -1,79 +1,47 @@
-open Result
+open V1
 let pf pp = Format.fprintf pp
 
-let pp_console_error pp = function
-  | `Invalid_console str -> pf pp "invalid console %s" str
-
-let unspecified pp m = pf pp "unspecified error - %s" m
+let pp_device_error pp = function
```

Comment:
The `_pp.pp_` is indeed pretty annoying. Two options:
- I can move the function to `Mirage_runtime`
- I can just drop the `pp_` prefix, and tell people to not open `Mirage_pp`.

I tend towards the later, but both options seems ok.